### PR TITLE
Small fixes to README.md

### PR DESCRIPTION
**Pull request by engineerchuan**
`n_channels` and `n_samples` will come up a lot.  We should put this in the Guidelines.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,7 +106,9 @@ whereas dimension names do not have this prefix (e.g. "a tensor of
 dimension (channel, time)")
 
 * `waveform`: a tensor of audio samples with dimensions (channel, time)
-* `sample_rate`: the rate of audio dimensions (samples per second)
+* `sample_rate`: the rate of audio samples per second (Hz)
+* `n_samples` : the number of time samples
```
Comment (vincentqb) [inline README.md]: Grammar says we need an "s", but we do `n_freq` and not `n_freqs`.

**Comment by vincentqb** [comment]
We should be careful about the distinction between samples (or "time"?) and frames. The latter is often a small group of samples.

File: README.md (hunk 1 above)
Comment (vincentqb) [reply README.md]: I prefer avoiding `*s` since it's shorter, and aligned with `n_freq` :)

**Comment by engineerchuan** [comment]
Sorry - what changes did you want me to make here?  Or have you already merged this.

**Comment by vincentqb** [comment]
> Sorry - what changes did you want me to make here? Or have you already merged this.

Reverted my changes. This has not been merged.

**Comment by vincentqb** [comment]
#298 was merged.

**Pull request closed without merging**